odoo/http.py

```diff
@@ -1251,7 +1251,11 @@ def start_wrapped(status, headers):
             req = werkzeug.wrappers.Request(environ)
             root.setup_session(req)
             if req.session and req.session.debug and not 'wkhtmltopdf' in req.headers.get('User-Agent'):
-                new_headers = [('Cache-Control', 'no-cache')]
+
+                if "assets" in req.session.debug and ".js" in req.base_url:
```

Comment:
why not `"assets" == req.session.debug`?

Reply:
probably to support debug=assets,test to load test asset and be able to run tour. By default all tour of test are no more loaded to the visitor.

Reply:
oh ok :)